### DISCUSSION TITLE
Dynamic Redirect URI

### DISCUSSION THREAD

**Discussion by willsawyerrrr** [fief-dev/fief#393] [Ideas]
My team's frontend development workflow includes the automated creation and deployment of feature branches to a subdomain corresponding to their name - something like `https://<feature-branch-name>.example.com`.

Currently, when we wish to utilise our Fief deployment for authentication on a new feature branch deployment, we must open the Fief admin dashboard and add the new deployment's URL to the client's allowed redirect URIs.

It would be great to be able to specify dynamic (for example, wildcard or regex) redirect URIs so that we can add `https://*.example.com/auth/callback` once to allow all of our feature branches.

**Comment by fief-bailiff** [comment]
Hail, @willsawyerrrr 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by frankie567** [comment]
Unfortunately, for security reasons, it would be quite dangerous to allow wildcard redirect URIs.

In your case, I would recommend to use the Admin API in your workflow to add the redirect URI: https://example.fief.dev/admin/api/docs#/Clients/clients_update_clients__id__patch
